YarnSpinner.LanguageServer.Tests/ReferenceTests.cs

```diff
@@ -48,7 +48,7 @@ public async Task Workspace_FindsReferencesToNodes()
             {
 
                 node1.References.Should().HaveCount(2);
-                node1.Jumps.Should().HaveCount(1);
+                node1.Jumps.Should().HaveCount(2);
```

Comment:
Not really related to this PR, but figured a new PR would be overkill.
Test was failing because a second `jump` was added to this node in the JumpsAndDetours test file in the most recent commit.
<img width="3770" height="866" alt="{C2A04495-9F0F-410B-BC5E-F459BAE94595}" src="https://github.com/user-attachments/assets/070952ca-b9bf-47c1-bd54-32db50092436" />